libr/util/w32-sys.c

```diff
@@ -74,19 +73,6 @@ R_API char *r_sys_cmd_str_w32(const char *cmd) {
 
 	CreateChildProcess (cmd, out);
 
-	in = CreateFileA (argv0,
```

Comment:
Apparently this is used as a "FileExists" function ... Example:
!rabin2 xxx => CreateFileA("rabin2.exe", ...) which cannot work because rabin2.exe will never be in the current folder.
Nonsense, unless someone has a good explanation, I think this should definitely be removed.